### PR TITLE
`[Whitelisting]` Settings page design update

### PR DESCRIPTION
**Pull request by mudrila**

### CODE REVIEW

File: src/components/pages/DaoSettings/DAOGeneralSettingsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { useTranslation } from 'react-i18next';
+import NestedPageHeader from '../../ui/page/Header/NestedPageHeader';
+import { GeneralSettingsContainer } from './components/GeneralSettingsContainer';
+
+export default function DAOGeneralSettingsPage() {
```
[the remark this answers is not in the document]
Comment (mudrila) [reply src/components/pages/DaoSettings/DAOGeneralSettingsPage.tsx:5]: Yeah, agreed - though I was kinda following existing pattern in this PR
I'll have more changes to the settings page in the upcoming PR - are u ok if I'll address these renames in scope of that "upcoming" PR?

**Review by DarksightKellar — APPROVED** [review]

**Review by Da-Colon — COMMENTED** [review]
Any screenshots ready for viewing for these PRs?

**Comment by mudrila** [comment]
> Any screenshots ready for viewing for these PRs? 

Not quite finalised, will add in further PR

**Comment by mudrila** [comment]
@Da-Colon Have some screenshots with nearly finalized Settings Pages implementation here https://github.com/decentdao/decent-interface/pull/2434

**Review by adamgall — APPROVED** [review]

**Pull request merged**